lang/strings_english.txt

```diff
@@ -1193,6 +1193,7 @@ $s_total_time = 'Total time';
 $s_developer_stats = 'Developer Stats';
 $s_reporter_stats = 'Reporter Stats';
 $s_orct = '(open/resolved/closed/total)';
+$s_summary_header = '(open/resolved/closed/total/resolved ratio/ratio)';
```

Comment:
Since we're introducing a new language string, I see no reason to maintain the legacy format with the surrounding parentheses, which makes processing the string more complex than it needs to be (preg_split with PREG_SPLIT_NO_EMPTY flag, or explode + substr like in the graph API).

I suggest using a simple delimited string, so a simple _explode()_ call can do the job.